src/bandersnatch/main.py

```diff
@@ -156,7 +162,9 @@ async def async_main(args: argparse.Namespace, config: ConfigParser) -> int:
     elif args.op.lower() == "verify":
         return await bandersnatch.verify.metadata_verify(config, args)
     elif args.op.lower() == "sync":
-        return await bandersnatch.mirror.mirror(config, args.packages)
+        return await bandersnatch.mirror.mirror(
+            config, args.packages, not args.skip_root
```

Comment:
Lets do this in a dedicated PR - Seems scope creep for this PR title.

What's your use case here so I can just understand.

Reply:
Yes, will do